CONTRIBUTING.md

```diff
@@ -153,7 +153,9 @@ Adjust following parameters:
 ###### [Debugging TypeScript code](https://www.jetbrains.com/help/idea/running-and-debugging-typescript.html#ws_ts_debug_client_side_on_external_dev_server)
 
 - create a new `JavaScript Debug` configuration as described in the [Intellij documentation](https://www.jetbrains.com/help/idea/running-and-debugging-typescript.html#ws_ts_debug_client_side_on_external_dev_server)
-  - the targeted url is: http://localhost:10001/
+  - the targeted url is: 
+    - For `npm run start` or `npm run test:e2e`: http://localhost:10001/
```

Comment:
with the current implementation port should be `10002`
I would prefer to keep the original doc and explain how to configure the port